src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -34,13 +35,17 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        User user = userService.findByEmail(authentication.getName()).get();
+        Long userId = user.getId();
         ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
         return orderResponseDtoMapper.mapToDto(orderService.completeOrder(cart));
     }
 
     @GetMapping
-    public List<OrderResponseDto> getOrderHistory(@RequestParam Long userId) {
+    public List<OrderResponseDto> getOrderHistory(Authentication authentication) {
+        User user = userService.findByEmail(authentication.getName()).get();
+        Long userId = user.getId();
         return orderService.getOrdersHistory(userService.get(userId))
```

Comment:
```suggestion
        return orderService.getOrdersHistory(userService.get(user.getId()))
```
dont use redundant variables